JNWCollectionView/JNWCollectionViewFramework.m

```diff
@@ -131,9 +137,23 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
 	return self;
 }
 
+#pragma mark Notifications
+
+- (void)preferredScrollerStyleDidChangeNotification:(NSNotification *)note {
+	NSView *documentView = self.documentView;
+	
+	if (!CGSizeEqualToSize(NSSizeToCGSize(documentView.frame.size), NSSizeToCGSize(self.contentSize))) {
+		CGRect documentFrame = documentView.frame;
+		documentFrame.size.width = self.contentSize.width;
+		documentView.frame = documentFrame;
```

Comment:
Does this still work if the collection view is resized after connecting a mouse?

Could it be replaced by invalidating the layout first, then calling `layoutDocumentView`?